app/src/common/shared/com/igalia/wolvic/ui/widgets/Windows.java

```diff
@@ -1033,6 +1041,27 @@ public void exitResizeMode() {
         }
     }
 
+    private final SharedPreferences.OnSharedPreferenceChangeListener mPreferencesListener = new SharedPreferences.OnSharedPreferenceChangeListener() {
+        @Override
+        public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
+            if (!Objects.equals(key, mContext.getString(R.string.settings_key_window_distance)))
+                return;
+
+            WindowWidget frontWindow = getFrontWindow();
+            frontWindow.getPlacement().translationZ = getFrontWindowTranslationZ();
+            updateViews();
+        }
+    };
+
+    private float getFrontWindowTranslationZ() {
+        float value = SettingsStore.getInstance(mContext).getWindowDistance();
```

Comment:
This method only used to change the Window's placement translationZ, right ? Wouldn't be better to define it as a setter method instead ?

Reply:
I don't have a strong opinion, so I'm ok with your original change.